ethereum/json-types/src/main/java/tech/pegasys/teku/ethereum/json/types/validator/SyncCommitteeSelectionProof.java

```diff
@@ -81,6 +83,10 @@ public String getSelectionProof() {
     return selectionProof;
   }
 
+  public BLSSignature getSelectionProofSignature() {
+    return BLSSignature.fromBytesCompressed(Bytes.fromHexString(getSelectionProof()));
```

Comment:
tempting to either split out a variable or make
```
Bytes getSelectionProofBytes() {
   return Bytes.fromHexString(getSelectionProof);
}
```
not as a performance thing, just as a readability thing... Potentially if were testing it'd make more sense.

Reply:
This is used on the follow-up PR. I'll keep it as is for now because we want a shortcut to the BLSSignature object to improve readability.